articles/contributing/samples.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Contributing samples to the Microsoft QDK
+description: Learn how to contribute sample code to the Microsoft Quantum Development Kit (QDK).
+author: cgranade
+ms.author: chgranad
+ms.date: 10/12/2018
+ms.topic: article
+uid: microsoft.quantum.contributing.samples
+---
+
+# Contributing Samples to the Quantum Development Kit
+
+If you're interested in contributing code to the [samples repository](https://github.com/Microsoft/Quantum), thank you for making the quantum development community a better place!
+
+## The Quantum Development Kit Samples Repository
+
+To help you prepare your contribution to help out as much as possible, it's helpful to take a quick look at how the samples repository is laid out:
+
+```plaintext
+microsoft/Quantum
+📁 samples/
+  📁 algorithms/
+    📁 chsh-game/
+      📝 CHSHGame.csproj
+      📝 Game.qs
+      📝 Host.cs
+      📝 host.py
+      📝 README.md
+     ⋮
+  📁 arithmetic/
+  📁 characterization/
+  📁 chemistry/
+   ⋮
+```
+
+That is, the samples in the [microsoft/Quantum repository](https://github.com/microsoft/Quantum) are broken down by subject area into different folders such as `algorithms/`, `arithmetic/`, or `characterization/`.
+Within the folder for each subject area, each sample consists of a single folder that collects everything a user will need to explore and make use of that sample.
+
+## How Samples are Structured
+
+Looking at the files that make up each folder, let's dive into the [`algorithms/chsh-game/`](todo:todo) sample.
```

Comment:
Don't forget to add the correct link here.

Reply:
Ah, thanks for catching that. Didn't have it right at hand and forgot to add it back.